tests/performance/src/test/java/org/mule/test/performance/util/AbstractPerformanceTestCase.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+
+package org.mule.test.performance.util;
+
+
+import org.mule.tck.junit4.AbstractMuleContextTestCase;
+
+import org.databene.contiperf.junit.ContiPerfRule;
+import org.junit.Rule;
+
+public class AbstractPerformanceTestCase extends AbstractMuleContextTestCase {
```

Comment:
Created profile to pass the timeout and ContiPerf property only when runing perf tests.